src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPartiesMapping.java

```diff
@@ -182,42 +196,38 @@ public static String getIndividualFullName(Entity entity) {
     public static String getIndividualPreferredHearingChannel(String hearingType,
                                                                         HearingSubtype hearingSubtype,
                                                                         HearingOptions hearingOptions) {
-        if (eitherNull(hearingType, hearingSubtype)) {
```

Comment:
instead of wrapping everything below in nonNull() 

we can keep this if check, but just return null, instead of throwing an exception

Reply:
Reverted, keeping the check and returning null if true.